### ISSUE TITLE
questions about how to test the sketch-rnn model

### ISSUE THREAD

**Issue by yemeijuan** [tensorflow/magenta#944]
I've trained the sketch-rnn model, but I do not know how to run the test to get the sketch.I hope you can tell me how to test.
And I got the training data,and I found "kL" has still been larger than 0.2 in the end, and "recon" become a negative.Is there a problem ? How should it be resolved?

**Comment by hardmaru** [comment]
Hi

KL loss should be positive and the reconstruction loss iusually becomes negative so it is normal.

For an example of how to load a trained TensorFlow model, please take a look at this IPython Notebook example:

https://github.com/tensorflow/magenta-demos/blob/master/jupyter-notebooks/Sketch_RNN.ipynb
